media-sound/subsonic-bin/subsonic-bin-6.1.6.ebuild

```diff
@@ -1,35 +1,33 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit eutils user
+inherit eutils
 
 MY_PN="${PN//-bin}"
 
 DESCRIPTION="Subsonic is a complete, personal media streaming solution"
 HOMEPAGE="http://www.subsonic.org/"
 SRC_URI="mirror://sourceforge/${MY_PN}/${PV}/${MY_PN}-${PV}-standalone.tar.gz"
 
-LICENSE="GPL-3"
+LICENSE=""
```

Comment:
There is nothing about the license on the site and in the distribution, but:
```
Subsonic is developed and maintained by Sindre Mehus
Terms of use | Subsonic uses licensed third-party libraries.
© Subsonic AS – All rights reserved 
```
There is no source on sourceforge since 2016-02-05 / v6.0

Reply:
You should probably ask them or see what other distros do, but at worst, we can do "all-rights-reserved".

Reply:
distros take this (https://repology.org/project/subsonic/versions) as "unfree" and "cutstom". Fix this to add "all-rights-reserve" to ebuild.